src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeWriter.cs

```diff
@@ -90,8 +90,65 @@ private static void InitUserSecrets(string projectPath)
             
             if (result.ExitCode != 0)
             {
+                Console.Write("FAILED\n");
                 throw new Exception("Error while running dotnet-user-secrets init");
             }
+            else
+            {
+                Console.Write("SUCCESS\n");
```

Comment:
How does this work if the caller has asked for a JSON output?

Reply:
shouldn't be written console then. I'll add some checks.

Reply:
will be adding a console logger which will be aware of the json output